lib/commons/dom/is-in-text-block.js

```diff
@@ -33,10 +34,16 @@ function getBlockParent(node) {
 
 /**
  * Determines if an element is within a text block
+ * With `noLengthCompare` true, will return if there is any non-space text outside
+ * widgets. When false, compares the length of non-widget text to widget text
+ *
  * @param  {Element} node [description]
+ * @param  {Object} options Optional
+ * @property {Bool} noLengthCompare
  * @return {Boolean}      [description]
  */
-function isInTextBlock(node) {
+function isInTextBlock(node, options) {
+  const { getRoleType } = axe.commons.aria;
```

Comment:
Ran into this error again: https://github.com/dequelabs/axe-core/issues/3281

Looks like I can work around it by importing directly from get-role-type, rather than from aria/index. Doing that.